lib/src/main/java/uk/gov/di/ipv/core/library/service/CiStorageService.java

```diff
@@ -5,30 +5,56 @@
 import com.amazonaws.services.lambda.model.InvokeRequest;
 import com.amazonaws.services.lambda.model.InvokeResult;
 import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
```

Comment:
I think, can discuss about exception retry and failure mechanism for all lambdas call.

Reply:
Yeah sure will be good to discuss retry logic.